src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -1,19 +1,47 @@
-﻿using SnapshotProject  = PKSim.Core.Snapshots.Project;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Utility.Extensions;
+using PKSim.Core.Model;
+using SnapshotProject = PKSim.Core.Snapshots.Project;
 using ModelProject = PKSim.Core.Model.PKSimProject;
 
-
 namespace PKSim.Core.Snapshots.Mappers
 {
    public class ProjectMapper : ObjectBaseSnapshotMapperBase<ModelProject, SnapshotProject>
    {
+      private readonly IExecutionContext _executionContext;
+
+      public ProjectMapper(IExecutionContext executionContext)
+      {
+         _executionContext = executionContext;
+      }
+
       public override SnapshotProject MapToSnapshot(ModelProject project)
       {
-         throw new System.NotImplementedException();
+         return SnapshotFrom(project, snapshot =>
```

Comment:
This is the new code here. Only implemented for the building blocks that have been defined as snapshot for now. This will change over time obviously as we add more export to snapshots